src/main/java/com/ericsson/eiffelcommons/subscriptionobject/SubscriptionObject.java

```diff
@@ -0,0 +1,85 @@
+package com.ericsson.eiffelcommons.subscriptionobject;
+
+import java.io.IOException;
+
+import org.json.JSONArray;
+import org.json.JSONObject;
+
+import com.ericsson.eiffelcommons.utils.Utils;
+
+import lombok.Getter;
+
```

Comment:
Unit tests for these methods should be added.

Reply:
Unittests for these methods are added

Reply:
ok.